frontend/src/App.tsx

```diff
@@ -135,6 +135,22 @@ export default class App extends React.Component {
     setInterval(() => (this.chainsCache = []), 10000); // Wipe sorted chains cache every 10 seconds
   }
 
+  private async updateMetadata() {
+    try {
+      const { uniqueAddressCount, spacePledged } = await fetchMetadata();
+
+      this.appUpdate({
+        spacePledged: parseInt(spacePledged, 10),
+        uniqueAddressCount: parseInt(uniqueAddressCount, 10),
+      });
+    } catch (error) {
+      // if data is missing components are not rendered inside Header.tsx
+      console.log(`Failed to fetch unique address count: ${error}`);
```

Comment:
Makes debugging easier:
```suggestion
      console.error(`Failed to fetch unique address count: ${error}`);
```